app/models/report/list_report.rb

```diff
@@ -16,7 +18,7 @@ def prep_query(rel)
       joins = []
       questions = []
 
-      rel = rel.select("responses.id AS response_id").order("response_id")
+      rel = rel.select("SQL_CALC_FOUND_ROWS responses.id AS response_id").order("response_id")
```

Comment:
Not sure about using calc_found_rows. See here: http://stackoverflow.com/a/4765919/2066866

Can we just limit to 1001, and if there are 1001 returned, show the first 1000 and say 'there were more than 1000 results'?